components/video-player.scss

```diff
@@ -0,0 +1,55 @@
+.VideoPlayer{
+  font-family: 'Roboto';
+	position: relative;
+  max-width: max-content;
+
+	&__button{
+		width: 51px;
+		height: 51px;
+		border-radius: 50%;
+		background-color: #ffffff;
+		box-shadow: 0px 0px 16px 2px #69FFB3;
+		display: inline-block;
+		overflow: hidden;
+		cursor: pointer;
+	}
+
+	&__text{
+		position: relative;
+		display: inline-block;
+		font-size: 13px;
+		color: #ffffff;
+    top: -21px;
+    padding-left: 17px;
+	}
+
+	&__button-arrow{
+    position: relative;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%,-50%);
+	}
+}
+
+
+@media only screen and (max-width: 576px) {
```

Comment:
So, for media queries we got a utility for that, just import `@import 'include-media/dist/_include-media.scss';` and use it according to the [documentation](https://www.npmjs.com/package/include-media). You can define a new breakpoint. This one is fine like this, but in the future might be easier.